src/main/java/org/spongepowered/api/entity/living/Living.java

```diff
@@ -110,6 +111,43 @@ default DamageableData getMortalData() {
     }
 
     /**
+     * Gets the volume that the sounds for this entity are played
+     *
+     * @return The volume of the sounds for this entity
+     */
+    double getVolume();
+
+    /**
+     * Gets the pitch for the sounds for this entity
+     * (e.g for if it's a baby)
+     *
+     * @return The pitch of the sounds for this entity
+     */
+    double getPitch();
+
+    /**
+     * Gets the {@link SoundType} that is played when this entity gets hurt.
+     *
+     * @return The {@link SoundType} that is played when this entity gets hurt
+     */
+    SoundType getHurtSound();
+
+    /**
+     * Gets the {@link SoundType} that is played when this entity dies.
+     *
+     * @return The {@link SoundType} that is played when this entity dies
+     */
+    SoundType getDeathSound();
+
+    /**
+     * Gets the {@link SoundType} that is played when this entity falls
+     * This return a big or small fall sound based on how far the entity is specified to fall.
+     * @param distance How far the entity fell
+     * @return The {@link SoundType} that is played when this entity falls that distance
+     */
+    SoundType getFallSound(int distance);
```

Comment:
Yes, if you have noticed vanilla sound types. There is a small fall and a big fall.